ddtrace/internal/ci_visibility/_api_client.py

```diff
@@ -372,6 +381,12 @@ def fetch_settings(self) -> TestVisibilityAPISettings:
                 )
             else:
                 early_flake_detection = EarlyFlakeDetectionSettings()
+
+            quarantine = QuarantineSettings(
+                enabled=attributes.get("quarantine", {}).get("enabled", False)
+                or asbool(os.getenv("_DD_TEST_FORCE_ENABLE_QUARANTINE"))  # ꙮ
```

Comment:
This is here to make my life easier to test quarantine while we don't have the backend, we don't need to keep it (or we can move it somewhere else).